src/pages/roommates.js

```diff
@@ -1,33 +1,74 @@
-import RoommateGrid from '@/components/RoommateGrid';
-import RoommateFilter from '@/components/RoommatesFilter';
-import Head from 'next/head';
-import { getAllUsers } from '@/firebase/userDb';
-import { useAuthUser } from '@/firebase/auth';
+import RoommateGrid from "@/components/RoommateGrid";
+import RoommateFilter from "@/components/RoommatesFilter";
+import Head from "next/head";
+import { getAllUsers } from "@/firebase/userDb";
+import { useAuthUser } from "@/firebase/auth";
+import { useState } from "react";
 
 export default function Roommates(props) {
-	const [user, loading] = useAuthUser();
-	// do not show current user in grid if they are logged in
-	const users = (user == null) 
-		? props.users 
-		: props.users.filter((u) => u.uid !== user.uid);
-	return (
-		<>
-			<Head>
-				<title>Roommate Suggestions</title>
-			</Head>
-			<div style={{ display: 'flex', padding: '10px 0px 0px 20px' }}>
-				<RoommateFilter />
-				<RoommateGrid users={users} />
-			</div>
-		</>
-	);
+  const [user, loading] = useAuthUser();
+  const [FilterRoommates, setFilterRoommates] = useState([0, [], []]);
+
+  // do not show current user in grid if they are logged in
+  let users =
+    user == null ? props.users : props.users.filter((u) => u.uid !== user.uid);
+
+  if (FilterRoommates[1].length > 0) {
+    users = FilterRoommates[1];
+  }
+
+  return (
+    <>
+      <Head>
+        <title>Roommate Suggestions</title>
+      </Head>
+      <div style={{ display: "flex", padding: "10px 0px 0px 0px" }}>
```

Comment:
Looks like you didn't change the style for RoommateFilter but why is its width so much shorter than the PropertyFilter? Is it because of the 20px left padding here?

Reply:
Could you try to make its style consistent with the PropertyFilter?

Reply:
For the first comment, as I wrote in the pull request description: 2. Max number of users to display in each section is 10 users, that's why it's not showing all users...

Reply:
My bad, didn't read that.

Reply:
Updated the style to make it consistent with propertyFilter.